packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -385,19 +404,73 @@ export default class extends EntityModel {
   }
 
   async joinOrg(
```

Comment:
Appreciate this was here before, and I don't want us to rock the boat and change it right now. But I wonder if it should really be `Org.addUser` rather than `joinOrg`. I guess it depends on which way we think the authorization is generally going to go.

Reply:
It also depends on the implementation we got with for organization invitations I think. `Org.addUser` sounds like it should just work when you call it, whereas `joinOrg` is more like the user requesting to join an org, which could fail if they don't supply the correct invitation code.

But let's revisit this when we actually get around to [implementing organization invitation](https://app.asana.com/0/1202805690238892/1202980861294704/f)